tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/wifi.py

```diff
@@ -4,8 +4,8 @@
 
 from marionette_driver import expected, By, Wait
 from marionette_driver.errors import StaleElementException
-
```

Comment:
There is no PEP8 rule that requires that. I deleted it, but we can put it back. I'm fine with both